stellarphot/photometry/photometry.py

```diff
@@ -154,6 +158,11 @@ def single_image_photometry(ccd_image, sourcelist, camera, observatory_location,
     the `use_coordinates` parameter should be set to "sky".
     """
 
+    def msgNexit(msg):
```

Comment:
I'm not 100% convinced that this function is need but it should definitely be outside the `single_image_photometry` body and be underscore-named e.g. `msg_and_exit`

Reply:
I removed the inner function, I suppose it is a matter of style.